apps/web/components/shared/kyc-4-upload.tsx

```
@@ -0,0 +1,465 @@
+          <div 
+            className={`
+              border-2 border-dashed rounded-lg p-8 text-center 
+              transition-all duration-300 ease-in-out transform 
+              hover:scale-[1.02] hover:border-black hover:bg-gray-50 
+              hover:shadow-lg cursor-pointer relative
+              ${isProcessing ? 'opacity-50 pointer-events-none' : ''}
+            `}
+            onDrop={handleDrop}
+            onDragOver={(e: React.DragEvent) => e.preventDefault()}
+            onClick={() => !isProcessing && document.getElementById('file-upload')?.click()}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)